src/util/replaygain.h

```diff
@@ -35,6 +36,15 @@ class ReplayGain final {
             "http://wiki.hydrogenaud.io/index.php?title=ReplayGain_2.0_specification#Peak_amplitude: "
             "The maximum peak amplitude value is stored as a floating number, "
             "where 1.0 represents digital full scale");
+#else
```

Comment:
yeah this is what I want to avoid.  I'd prefer to not use constexpr at all in Mixxx until Microsoft catches up.  Let's go back to plain "const" for now, even though it's not quite the same meaning.